### ISSUE TITLE
[MYSQL] Import schema for auto-generated column fails after Ora2Pg commit

### ISSUE THREAD

**Issue by shubham-yb** [yugabyte/yb-voyager#355]
This is the original ticket
[#315](https://github.com/yugabyte/yb-voyager/issues/315)

So an issue was put up on Ora2Pg and they implemented migration of auto-generated columns using triggers.
[Commit](https://github.com/darold/ora2pg/commit/59028d196fbde077af7f158b3afb33ebe54f0e2a)

But now the `import-schema` fails with error.

**Original schema:**

```
CREATE TABLE auto_price (
	id INT,
	first_name VARCHAR(50),
	gender VARCHAR(50),
	price_dollar DECIMAL(7,2),
    price_pound DECIMAL(7,2) GENERATED ALWAYS AS (price_dollar * 0.22) STORED
);
```

**Previously exported schema:**

```
CREATE TABLE auto_price (
	id bigint,
	first_name varchar(50),
	gender varchar(50),
	price_dollar decimal(7,2),
	price_pound decimal(7,2)
) ;
```

**Exported schema after commit:**

```
CREATE TABLE auto_price (
	id bigint,
	first_name varchar(50),
	gender varchar(50),
	price_dollar decimal(7,2),
	price_pound decimal(7,2)
) ;
DROP TRIGGER IF EXISTS virt_col_auto_price_trigger ON auto_price CASCADE;

CREATE OR REPLACE FUNCTION fct_virt_col_auto_price_trigger() RETURNS trigger AS $BODY$
BEGIN
	NEW.price_pound = (NEW.price_dollar * 0.22);

RETURN NEW;
end
$BODY$
 LANGUAGE 'plpgsql' SECURITY DEFINER;

CREATE TRIGGER virt_col_auto_price_trigger
        BEFORE INSERT OR UPDATE ON auto_price FOR EACH ROW
        EXECUTE PROCEDURE fct_virt_col_auto_price_trigger();


```

**Error while importing:**

```
importing      TABLE      
    ERROR: unterminated dollar-quoted string at or near "$BODY$ BEGIN 	NEW.price_pound = (NEW.price_dollar * 0.22); " (SQLSTATE 42601)
    STATEMENT: CREATE OR REPLACE FUNCTION fct_virt_col_auto_price_trigger() RETURNS trigger AS $BODY$
BEGIN
	NEW.price_pound = (NEW.price_dollar * 0.22);
ERROR: unterminated dollar-quoted string at or near "$BODY$ BEGIN 	NEW.price_pound = (NEW.price_dollar * 0.22); " (SQLSTATE 42601)
```

**Comment by amit-yb** [comment]
@shubham-yb Looks like @sanyamsinghal fixed this issue as part of https://github.com/yugabyte/yb-voyager/pull/399 . Please, verify and close this issue.

**Comment by shubham-yb** [comment]
@amit-yb @sanyamsinghal I'm still getting the issue.



![Image](https://user-images.githubusercontent.com/65844208/196113706-9a4b7e93-f7c0-4a6d-88d0-e401d695ca37.png)

**Comment by sanyamsinghal** [comment]
Fix: change the parsing logic to lookahead parsing instead of based on kind of sql file

**Comment by sanyamsinghal** [comment]
@shubham-yb can you please verify this bug with main branch now. Changes merged in #506
